features/steps/cli_steps.py

```diff
@@ -407,18 +408,16 @@ def update_pyproject_toml(context: behave.runner.Context, new_source_dir):
 
 @given("I have updated kedro requirements")
 def update_kedro_req(context: behave.runner.Context):
-    """Replace kedro as a standalone requirement with a line
-    that includes all of kedro's dependencies (-r kedro/requirements.txt)
-    """
+    """Remove kedro as a standalone requirement."""
     reqs_path = context.root_project_dir / "src" / "requirements.txt"
-    kedro_reqs = f"-r {context.requirements_path.as_posix()}"
 
     if reqs_path.is_file():
         old_reqs = reqs_path.read_text().splitlines()
         new_reqs = []
         for req in old_reqs:
-            if req.startswith("kedro"):
-                new_reqs.append(kedro_reqs)
+            if req.startswith("kedro") and Requirement(req).name.lower() == "kedro":
```

Comment:
The logic before was to replace `kedro` with all `kedro` dependencies. In a way, something like `pip install kedro --dependencies-only` (which doesn't exist I believe).

Now I'm just removing kedro, because this is only for our tests and the `behave` environment has the development version of kedro preinstalled already.

Reply:
Also I made one improvement: parsing the requirement name so we are only dropping `kedro` and not, say, `kedro-datasets` or `kedro-airflow`. This was incorrect before.